FindEMBEDDED.cmake

```diff
@@ -0,0 +1,7 @@
+# - Try to find embedded platforms (RPI/IMX6)
+# Once done this will define
+#
+#  EMBEDDED_FOUND        - system is RPI / IMX6 and we most probably want to compile for GLES2 support
+#                          (don't configure for OpenGL)
+
+find_path(EMBEDDED_FOUND NAMES linux/imxfb.h bcm_host.h HINTS /opt/vc/include)
```

Comment:
yes, thats most probably correct, good catch. Maybe also checking if target arch(CMAKE_SYSTEM_PROCESSOR) is arm might be enough.